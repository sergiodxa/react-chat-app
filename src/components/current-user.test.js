import React from "react";
import { render } from "@testing-library/react";

import CurrentUser from "./current-user";

test("CurrentUser component", () => {
  const { asFragment } = render(<CurrentUser username="test-user" />);
  expect(asFragment()).toMatchSnapshot();
});
