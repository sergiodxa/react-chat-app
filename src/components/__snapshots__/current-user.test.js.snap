// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`CurrentUser component 1`] = `
<DocumentFragment>
  .emotion-2 {
  display: -webkit-box;
  display: -webkit-flex;
  display: -ms-flexbox;
  display: flex;
  padding: 1rem .5rem;
  -webkit-align-items: center;
  -webkit-box-align: center;
  -ms-flex-align: center;
  align-items: center;
}

.emotion-0 {
  background-color: #DCDCDC;
  border-radius: .25rem;
  height: 2rem;
  margin: 0;
  margin-right: .25rem;
  padding: 0;
  width: 2rem;
}

.emotion-1 {
  font-weight: 500;
}

<div
    class="emotion-2"
  >
    <figure
      class="emotion-0"
    />
    <strong
      class="emotion-1"
    >
      test-user
    </strong>
  </div>
</DocumentFragment>
`;
